### DISCUSSION TITLE
How do I place a large legend (key) outside of the mesh?

### DISCUSSION THREAD

**Discussion by ebardie** [plotters-rs/plotters#657] [Q&A]
I'd like to place the legend to the right of the mesh so as not to cover the plot series.

I've tried using `.position(SeriesLabelPosition::Coordinate(x,y)`  but that ends up with the markers pinning the legend to the side of the mesh:

![2024-10-29_19-14](https://github.com/user-attachments/assets/af804726-aa49-458f-879e-88f103396994)

Any idea if/how it's possible to move the legend outside the mesh?

**Comment by jmcelroy01** [comment]
Hello, 
I was able to do this using the `split_horizontally` function: 

```
    let root = SVGBackend::new("plot.svg", (1900, 1000)).into_drawing_area();
    root.fill(&WHITE)?;
    let (chart_area, legend_area) = root.split_horizontally(1200);
```

**Reply by ebardie** [reply]
Sorry @jmcelroy01, but I still don't see how to get `ChartBuilder` , `ChartContext` , or whomever, to place the legend in a different drawing area to the mesh.
